Applications/AlexNet/jni/meson.build

```diff
@@ -1,8 +1,15 @@
-build_root = meson.build_root()
-res_path = meson.current_source_dir() / '..' / 'res'
+fs = import('fs')
+app_res_dir = fs.parent(meson.current_source_dir()) / 'res'
+build_app_res_dir = nntr_app_resdir / 'AlexNet'
 
-nntr_alex_resdir = nntr_app_resdir / 'AlexNet'
-run_command('cp', '-lr', res_path, nntr_alex_resdir)
+if host_machine.system() == 'windows'
```

Comment:
You don't need that, it can be done in native file.
Please see https://mesonbuild.com/Machine-files.html#binaries
and also my [POC](https://github.com/piotrrak/nntrainer-testsuite-deps-poc/blob/master/win32.ini)
```ini
[binaries]
#mkdir = ['mkdir']
mkdir = ['wine', 'cmd.exe', '/c','mkdir']
nproc = ['wine', 'cmd', '/c', '@echo', '%NUMBER_OF_PROCESSORS%']
touch = ['@DIRNAME@'/'touch.bat']
```
It'll require `find_program('cp')`